src/shared/components/person/notifications.tsx

```diff
@@ -514,12 +474,13 @@ export class Notifications extends Component<
     await i.refetch();
   }
 
-  async handleMessageTypeChange(i: Notifications, event: any) {
-    i.setState({
-      messageType: event.target.value as NotificationDataType,
+  async handleMessageTypeChange(val: NotificationDataType) {
+    this.setState({
+      messageType: val,
       cursor: undefined,
     });
-    await i.refetch();
+    // TODO: somehow it cannot refetch from here
+    this.refetch();
```

Comment:
This one is not working, all API requests from here are failing with `CORS: missing origin`. Other places use `linkEvent` but that doesnt work here as the param is not an event.

Reply:
See comment above.